.github/workflows/php.yml

```diff
@@ -12,11 +12,55 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.4, 8.0, 8.1]
+        php: [8.0, 8.1]
+        laravel: [6.*, 7.*, 8.*, 9.*]
+
+    name: build (${{ matrix.php }} w/ Laravel ${{ matrix.laravel }}
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+
+      - name: Cache dependencies
+        uses: actions/cache@v2.1.7
+        with:
+          path: ~/.composer/cache/files
+          key: dependencies-laravel-${{ matrix.laravel }}-php-${{ matrix.php }}-composer-${{ hashFiles('composer.json') }}
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+
+      # Installing a specific version of Laravel will force us to compatible versions of `illuminate/http` and
+      # `illuminate/support`. If we didn't do this step then we'd always only ever test on the latest versions of those
+      # two packages.
+      - name: Install dependencies
+        run: |
+          composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
+          composer update --prefer-stable --prefer-dist --no-interaction --no-suggest
+
+      - name: Check code standards
+        run: composer run-script lint
+
+      # - name: Execute static analysis
+      #   run: composer run-script psalm
+
+      - name: Run tests
+        run: composer run-script phpunit
+
+  test-php74:
```

Comment:
We had to rip PHP 7.4 out into its own test suite here because Laravel 9 is only supported on PHP 8.